src/tests/LitePsmJob-integration.t.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity 0.8.13;
+
+import "./DssCronBase.t.sol";
+
+import {LitePsmJob} from "../LitePsmJob.sol";
+import {LitePsmLike} from "../LitePsmJob.sol";
+
+interface DaiAbstract {
+    function approve(address, uint256) external;
+    function balanceOf(address) external view returns (uint256);
+}
+
+interface GemLike {
+    function approve(address, uint256) external;
+    function balanceOf(address) external view returns (uint256);
+    function decimals() external view returns (uint8);
+    function transfer(address, uint256) external;
+    function transferFrom(address, address, uint256) external;
+}
+
+interface VatLike {
+    function frob(bytes32, address, address, address, int256, int256) external;
+    function hope(address) external;
+    function slip(bytes32, address, int256) external;
+    function ilks(bytes32) external view returns (uint256, uint256, uint256, uint256, uint256);
+    function debt() external view returns (uint256);
+    function Line() external view returns (uint256);
+    function urns(bytes32, address) external view returns (uint256, uint256);
+    function file(bytes32 ilk, bytes32 what, uint data) external;
+    function file(bytes32 what, uint data) external;
+}
+
```

Comment:
Please double-check for methods declared in the inline interfaces that are not being used in this file.

Reply:
everything was cleaned up in [092a8cd](https://github.com/makerdao/dss-cron/pull/25/commits/092a8cdcd7eb3470b95287314b863123169eb59e)